Makefile

```diff
@@ -43,8 +52,16 @@ brokerpak-user-docs.md: *.yml
 	docker run $(DOCKER_OPTS) \
 	$(CSB) pak docs /brokerpak/$(shell ls *.brokerpak) > $@
 
+.PHONY: examples
+examples: build ## display available examples
+	docker run $(DOCKER_OPTS) \
+	-e SECURITY_USER_NAME \
+	-e SECURITY_USER_PASSWORD \
+	-e USER \
+	$(CSB) client examples
+
 .PHONY: run-examples
-run-examples: build
+run-examples: build ## run examples against CSB on localhost (run "make run" to start it), set service_name and example_name to run specific example
```

Comment:
Actually, `service_name and example_name to run specific example` is not accurate as those parameters are not being passed down in the actual command. We need to do 

`$(CSB) client run-examples --service-name=$(service_name) --example-name=$(example_name) -j $(PARALLEL_JOB_COUNT)`